spec/ParseQuery.Aggregate.spec.js

```diff
@@ -401,11 +401,13 @@ describe('Parse.Query Aggregate testing', () => {
           const query = new Parse.Query(TestObject);
           return query.aggregate(pipeline);
         })
-        .then(done.fail)
-        .catch(error => {
-          expect(error.code).toEqual(Parse.Error.INVALID_QUERY);
+        .then(results => {
+          const counts = results.map(result => result.count);
+          expect(counts.length).toBe(3);
+          expect(counts.sort()).toEqual([1, 2, 4]);
```

Comment:
If group by date works, wouldn't the results be obj1 + obj3 is one group and obj2 is another.

count would be [1, 2] what does the 4 come from?

Reply:
4 come from the default data that is added before each test. It returns [ objectId: undefined, count: 4]

Reply:
This data here: https://github.com/parse-community/parse-server/blob/master/spec/ParseQuery.Aggregate.spec.js#L21